srv/components/provisioner/files/cluster_sls.template

```diff
@@ -94,7 +94,8 @@ cluster:
     storage:
       enclosure: enclosure-{{ (node_num + 1) }}
       cvg:
-        - data_devices:
+        - cvg_name:
+          data_devices:
```

Comment:
The correct template should be:
cvg:
    - cvg_name
    - data_devices
    - metadata_devices

Reply:
It's correct, cvg itself is a list of cvgs, so convention is correct, will change if I find it wrong in the testing.